src/Core/components/ControlScheme.tsx

```diff
@@ -131,8 +140,13 @@ interface IProps {
 
 interface IState {
   functionality: any
+  controlPreviewModal: boolean
+  image: string
 }
```

Comment:
The data structure of this component is really confusing and needs to be addressed in the future.